app/js/app.jsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import {createHistory, useBasename} from 'history';
```

Comment:
Most acceptable.

In reality, I'll be fixing the lint errors that currently happen because linting wasn't initially configured for this project.  *...until john updates the lint file...*